components/sync/driver/brave_sync_service_impl_unittest.cc

```diff
@@ -180,4 +186,22 @@ TEST_F(BraveSyncServiceImplTest, ValidPassphraseKeyringLocked) {
 
 #endif  // defined(OS_APPLE)
 
+TEST_F(BraveSyncServiceImplTest, DISABLED_EmulateGetOrCreateSyncCodeCHECK) {
```

Comment:
Maybe we can try `EXPECT_EXIT` on this case instead of disabling it.

Reply:
thanks, I am trying to redo with your advise

Reply:
this is fixed in https://github.com/brave/brave-core/pull/11935/commits/732c4a5467171d7ec63051d993b0b888e209c5d2